MIT_GEOS5PlugMod/CMakeLists.txt

```diff
@@ -0,0 +1,118 @@
+esma_set_this ()
+
+set (SRCS
+  MIT_GEOS5PlugMod.F90
+  )
+
+if (NOT DEFINED MIT_CONFIG_ID)
+   message("You need to specify MIT_CONFIG_ID")
+   return()
+endif() 
+
+SET (MITGCM_ROOTDIR ${CMAKE_CURRENT_SOURCE_DIR}/@mit)
+SET (MITGCM_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR}/mitgcm_setup)
+SET (MITGCM_CONFIG_DIR ${CMAKE_CURRENT_SOURCE_DIR}/configs/${MIT_CONFIG_ID})
+
+#SET (MITLIBS ${MITGCM_BUILD_DIR}/code_split_driver/libmitgcmuv_driver.a)
+if(CMAKE_Fortran_COMPILER_ID MATCHES "GNU")
+     SET (MITGCM_BUILD_OPTIONS build_options.gnu)
+     set (FC gfortran)
+     set (CC gcc)
+elseif(CMAKE_Fortran_COMPILER_ID MATCHES "Intel")
+     SET (MITGCM_BUILD_OPTIONS build_options.intel)
+     # Path to Intel compiler to overwrite defaults for mpif90
+     set (FC ifort)
+     set (CC icc)
+endif()
+
+if(CMAKE_BUILD_TYPE MATCHES "Debug")
+     SET (MITGCM_EXTRA_BUILD_OPTIONS "-ieee")
+elseif()
+     SET (MITGCM_EXTRA_BUILD_OPTIONS "")
+endif()
+
+
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/mitgcm_setup
+     DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
+
+file(COPY ${MITGCM_CONFIG_DIR}/code
+     DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/mitgcm_setup)
+
+#file (GLOB OBJS CONFIGURE_DEPENDS ${MITGCM_BUILD_DIR}/build/*.o
```

Comment:
Why aren't commented lines removed?

Reply:
The comments are a reminder of older syntax from mit that was changed here by @atrayano. it is a way to retain the connection to their syntax (if we need to sync with them again)